it/uk/gov/hmrc/agentsubscription/controllers/SubscriptionAuditingSpec.scala

```diff
@@ -45,7 +44,33 @@ class SubscriptionAuditingSpec extends BaseAuditSpec with Eventually with DesStu
     }
   }
 
+  "updating a partial subscription" should {
+    "audit an AgentSubscription event" in {
+      writeAuditMergedSucceeds()
+      writeAuditSucceeds()
+
+      requestIsAuthenticated().andIsAnAgent().andHasNoEnrolments()
+      agentRecordExists(utr, true, arn)
+      allocatedPrincipalEnrolmentNotExists(arn)
+      deleteKnownFactsSucceeds(arn)
```

Comment:
I thought we are just checking the known fact is the same, not overriding it with new known fact